manim/mobject/geometry/arc.py

```diff
@@ -641,11 +640,11 @@ def construct(self):
 
     def __init__(
         self,
-        point=ORIGIN,
+        point: list | np.ndarray | None = ORIGIN,
```

Comment:
The optional annotation was wrong, point cannot actually be `None`.

```suggestion
        point: list | np.ndarray = ORIGIN,
```